.rubocop.yml

```diff
@@ -61,6 +61,11 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+# in rspec, expect{...} is conventional
```

Comment:
Obviously, there are no cases because the rubocop/travis rules would fail the build on it.  So we don't need to grep to know that.  Otherwise, we wouldn't be having this conversation.  

When I'm reading specs, differentiation between paren and curly following `expect` is a non-issue.  It's not something I need to highlight.  But the difference between `method_a` and `method_b` is.  The formatting guideline comes from writing ruby blocks and hashes, not this particular invocation inside rspec DSL.  

As for extending community standards, how about starting with Blacklight where AllCops are excluded for all specs?  Here I'm covering concretely failing methods and correcting expectations that were never once legitimate.  I'm not going to go to war over whitespacing two lines, but this is kind of thing where it feels trifling.